src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -485,7 +485,8 @@ int32_t viewport_interaction_right_click(int32_t x, int32_t y)
 static void viewport_interaction_remove_scenery(TileElement* tileElement, int32_t x, int32_t y)
 {
     auto removeSceneryAction = SmallSceneryRemoveAction(
-        x, y, tileElement->base_height, tileElement->AsSmallScenery()->GetSceneryQuadrant(),
+        { x, y, tileElement->base_height << 3 },
```

Comment:
use * 8 for this. In general we are trying to move away from << >>. There are certain situations that this can't be done (some rendering functions) but this is not one of them.